runtime/doc/eval.txt

```diff
@@ -2007,6 +2007,7 @@ range( {expr} [, {max} [, {stride}]])
 readfile( {fname} [, {binary} [, {max}]])
 				List	get list of lines from file {fname}
 reltime( [{start} [, {end}]])	List	get time value
+reltimefloat({time})		Float	turn the time value into a Float
```

Comment:
The space after the paren in this list of functions is intentional so that `ctrl-]` works. 

    reltimefloat( {time})